packages/design-system/src/components/Alert/Alert.stories.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
```

Comment:
I refactored Alert to see what would be involved if we used SB's new [Component Story Syntax](https://storybook.js.org/docs/react/api/csf). Discovered some features we currently use were either moved to new packages marked for deprecation (`useArgs from @storybook/client-api`), or aren't recommended patterns (`Template.bind()`)